tests/fixtures/datastore_factory.py

```diff
@@ -507,15 +508,10 @@ def make_datastore(
                     if output_path != sub_cache_path:
                         warnings.warn(f'cache path {sub_cache_path} does not match output path {output_path}')
 
-            # make sure that the aligned images get into the cache, too
-            if (    use_cache and
-                    'cache_name_ref' in locals() and
-                    os.path.isfile(os.path.join(cache_dir, cache_name_ref)) and
-                    'cache_name_new' in locals() and
-                    os.path.isfile(os.path.join(cache_dir, cache_name_new))
-            ):
-                for im in ds.sub_image.aligned_images:
-                    copy_to_cache(im, cache_dir)
+            # save the aligned images to cache
```

Comment:
Don't you still want an if around this checking (at least) `use_cache` ?

Reply:
yes, you are right!